docs/1.1-InstallationPrerequisites.md

```diff
@@ -216,4 +216,19 @@ The ConfigMap could be generated and deployed with the `hack/scripts/set_operato
 
     .hack/scripts/set_operator_configmap.sh -a ${ACCOUNT_LIMIT} -v ${VCPU_QUOTA} -r "${OSD_STAGING_1_OU_ROOT_ID}" -o "${OSD_STAGING_1_OU_BASE_ID}"
 
+### 1.1.5 Environment Variables
+Now we have all required Roles defined, set up your required environment variables:
+```bash
+export AWS_PAGER= # This is set so that it doesn't page out to less and block integration testing
```

Comment:
There is no info on what to put on AWS_PAGER. Is it actually optional?

Reply:
I don't even have this set in my local setup, and did a search in and see it's not even used in the codebase. @iamkirkbater or @ArielLima can this just be removed?